src/views/Layout.jsx

```diff
@@ -1,18 +1,27 @@
 import { Outlet, NavLink } from 'react-router-dom';
+import { ReactSVG } from 'react-svg';
 
 export function Layout() {
 	return (
 		<>
 			<div className="Layout">
 				<header className="Layout-header">
-					<h1>PREDICT-A-BUY</h1>
+					<div className="title-container">
+						<ReactSVG className="star-top" src="/img/star4.svg" />
```

Comment:
I've never imported SVGs like this and boy have you opened my eyes! This is super neat because they are way easier to change the color of! (which is super useful for dark mode!)